core/filestorage/filesync/upload.go

```diff
@@ -163,7 +163,7 @@ func (s *fileSync) retryingHandler(ctx context.Context, it *QueueItem) (persiste
 			}
 		}
 		if limitErr == nil || !limitErrorIsLogged {
-			log.Error("retry uploading file error",
+			log.Debug("retry uploading file error",
```

Comment:
maybe just ignore 

`format.IsNotFound` and `strings.Contains(err.Error(), "failed to fetch all nodes")`?